test/typescript/multiple-stacks/test.ts

```diff
@@ -0,0 +1,106 @@
+/**
+ * Testing a full cycle of diff, deploy and destroy
+ *
+ * @group typescript
+ */
+
+import { TestDriver } from "../../test-helper";
+
+describe("full integration test", () => {
+  let driver: TestDriver;
+
+  beforeAll(() => {
+    driver = new TestDriver(__dirname)
+    driver.setupTypescriptProject()
+  });
+
+  test("synth", () => {
+    driver.synth()
+    expect(driver.synthesizedStack('first')).toMatchSnapshot()
+    expect(driver.synthesizedStack('second')).toMatchSnapshot()
+  });
+
+  test("synth with json output", () => {
+    expect(driver.synth('--json')).toMatchSnapshot()
+  });
+
+  test("diff", () => {
+    expect(driver.diff('first')).toMatchInlineSnapshot(`
+      "Stack: first
+      Resources
+       + NULL_RESOURCE       test                null_resource.test
+
+
+      Diff: 1 to create, 0 to update, 0 to delete.
+      "
+    `);
+
+
+    expect(driver.diff('second')).toMatchInlineSnapshot(`
+      "Stack: second
+      Resources
+       + NULL_RESOURCE       test                null_resource.test
+
+
+      Diff: 1 to create, 0 to update, 0 to delete.
+      "
+    `);
+
+
+    let error;
+    try { driver.diff() } catch(e) { error = e.message }
+    expect(error).toMatch('Found more than one stack, please specify a target stack first, second');
```

Comment:
Could probably simplified to something like `expect(() => driver.diff()).toThrowError('Found more than …')` [docs](https://jestjs.io/docs/expect#tothrowerror)